### PR TITLE
Add programatically generated changelogs.

### PR DESCRIPTION
**Pull request by jimconner**
I read the docs properly(ish) this time around, so here are some programitically generated change logs for review.

### CODE REVIEW

**Comment by orendain** [comment]
Since the changelog generator works by recognizing PRs with certain labels, all relevant PRs that get merged should be labelled.  https://github.com/orendain/DemeoMods/blob/6800880a550d789a848bdf91e19902409e7f1f8c/RELEASE.md#1-ensure-each-pr-is-properly-tagged

For example:
There's no changelog entry about the "Flipping Out" ruleset.  That's because the PR is unlabelled:
image

I've just modified that PR (adding labels and modifying title, as per release doc link above) to be:
image

If the changelog generator is run again, it should now pick up this PR.

What I've done in the past was open up the closed PRs page on GitHub and just scroll through, making sure all PRs are labelled (with at least `area` and a `type`) and have titles in the active voice.

**Comment by jimconner** [comment]
Fresh commit pushed with Flipping Out correcly included. Please re-review.

**Comment by orendain** [comment]
My last message was vague, allow me to rephrase to clear up possible confusion.  This is also probably due to my writing the `RELEASE.md` draft in one go and not reviewing it to make sure all the steps were clear.

TL;DR: We must skim through [the list of closed PRs](https://github.com/orendain/DemeoMods/pulls?q=is%3Apr+is%3Aclosed) and properly label them before running the changelog generator script.  Else, the changelog is incomplete (doesn't reflect all of the changes that were made).

---

Each changelog generator config (e.g. [config for core](https://github.com/orendain/DemeoMods/blob/main/HouseRules_Configuration/.github_changelog_generator)) includes:
- [A label](https://github.com/orendain/DemeoMods/blob/main/HouseRules_Configuration/.github_changelog_generator#L16).  Any PR labelled with that label gets automatically picked up and added to the generated changelog.
- [Another set of labels](https://github.com/orendain/DemeoMods/blob/main/HouseRules_Configuration/.github_changelog_generator#L10-L14). This defines the changelog sections (e.g., "Fixes", "Features/Enhancements") that the PR gets automatically added to.

In order for the generator to work, all merged PRs need to be properly labelled.  Else, they don't make it into the changelog.

**Currently, our closed list of PRs in GitHub looks like this:**

image

Notice that in the screenshot above:
- Many PRs are missing required labels (`area` and `type` labels).
- (less important) Some titles are vague (e.g., "Define stealth damage for abilities" doesn't mention it introduces a new rule).

**Compare this with PRs that made it into a previous release:**

image

---

I need to go back and update the proposed `RELEASE.md` doc to clear up vagueness.

**Review by orendain — APPROVED** [review]
LGTM
